### PR TITLE
css-positioning | ChiaYu Lin

### PR DESCRIPTION
**Pull request by cyulin17**
reletive-positioning

absolute-positioning

fixed-positioning

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
```
Comment (BrettAlbright) [inline css-positioning/absolute.css:33]: you are missing code here, you want to text to display in the same place as in the example

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+
+  /* your code here */
```
Comment (BrettAlbright) [inline css-positioning/absolute.css:41]: you need code here to make sure the overlay element is removed from the document flow and it has the same height and width as its parent to darken the image

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
a few changes!

File: css-positioning/absolute.css (hunk 2 above)
Comment (cyulin17) [reply css-positioning/absolute.css:41]: absolute-positioning

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**